reporters_db/data/reporters.json

```diff
@@ -15864,6 +15881,7 @@
             ],
             "name": "Ohio Circuit Court Reports",
             "variations": {
+                "Jahn": "Ohio C.C.",
```

Comment:
Where did this come from.  I cant find a `Jahn` in any of my big books of reporters

Reply:
I found that here: https://fclawlib.libguides.com/ohioprimarylaw/cases

OHIO REPORTER NAME | Ohio C.C.
Ohio Circuit Reports (Jahn) | Ohio C.C

I don't know exactly why Jahn, but based on that information, also I inferred it from this IA file and many others:https://ia903102.us.archive.org/14/items/law.free.cap.jahn.10/1.1503259.json

Official citatation in IA is "10 Jahn 1" and in https://api.case.law/v1/cases/1503259/ the official citation is: "10 Ohio C.C. 1"

With that i deduced that: "Jahn" -> "Ohio C.C."

Reply:
great.  Thanks